python/CMakeLists.txt

```diff
@@ -111,8 +111,8 @@ add_subdirectory(torch_mlir/eager_mode)
 # Required for running the update_torch_ods.sh and update_shape_lib.sh scripts.
 ################################################################################
 
-# TODO: renable once it build on macOS Intel / M1
-#add_subdirectory(torch_mlir/_torch_mlir_custom_op_example)
+#TODO: reenable custom op example
+# add_subdirectory(torch_mlir/_torch_mlir_custom_op_example)
```

Comment:
i would also remove all these TODOs. I think @silvasean is going to rework custom ops in the future and these may not make sense.